server/models/Transaction.js

```diff
@@ -80,7 +99,12 @@ export default (Sequelize, DataTypes) => {
           paymentProcessorFee: this.paymentProcessorFee,
           netAmountInGroupCurrency: this.netAmountInGroupCurrency,
           amountInTxnCurrency: this.amountInTxnCurrency,
-          txnCurrency: this.txnCurrency
+          txnCurrency: this.txnCurrency,
+
+          // expense related fields
+          expenseCategory: this.expenseCategory,
+          expenseIncurredAt: this.expenseIncurredAt,
+          expensePayoutMethod: this.expensePayoutMethod
```

Comment:
🤔

I think the right pattern for this is the one we already use elsewhere: do a joint request and create a virtual getter on the Expense model to only get the attributes you need in that context (e.g. `Expense.dataForTransaction()` ).
That way, the client can get the category for example in a constant manner: `transaction.expense.category`. You don't want to introduce another way to write the variable `expense.category` like you do here with `expenseCategory`.